### PR TITLE
failing test for issue #7

### PR DESCRIPTION
**Pull request by tylkomat**
module placed at the top instead of the bottom

### CODE REVIEW

File: test/ComponentInstallerTest.php [hunk 1]
```diff
@@ -571,4 +571,65 @@ public function testOnPostPackageUninstallCanRemovePackageArraysFromConfiguratio
...
+            if (! strstr($argument[1], 'Do not inject')) {
+                return false;
+            }
+
+            if (! strstr($argument[2], 'application.config.php')) {
+                return false;
+            }
+
+            return true;
+        }), 0)->willReturn(1);
```
Comment (weierophinney) [inline test/ComponentInstallerTest.php:612]: Shouldn't this be 2, not 1?

Comment (tylkomat) [inline test/ComponentInstallerTest.php:612]: Yes you are right, but then there is something else wrong. Because I took this part from another testcase and the `$modules` variable still holds two entries.

**EDIT:** No, 1 is correct, means add it to `application.config.php`

Comment (weierophinney) [reply test/ComponentInstallerTest.php:612]: For anybody else coming across this, here's the explanation for why `1` is correct.

The `$ask` array has three items in it. The first is the question "Please select which config file you wish to inject", and has no option assigned to it. The next two are `Do not inject`, which has index `0`, and `application.config.php`, which has index `1`. The code above this is ensuring we get all three entries, and then returns the value `1`, indicating the user selected `application.config.php`.

Basically, ignore the indices within the method argument verification; what matters are the indices when creating the arguments within the `ComponentInstaller` class itself.
